configure.ac

```diff
@@ -26,70 +26,36 @@ AC_CHECK_HEADERS([ctype.h dirent.h errno.h locale.h signal.h \
 # Check for library functions
 AC_CHECK_FUNCS([memset setlocale])
 
-# Initialise i18n
-AM_GNU_GETTEXT_VERSION([0.17])
-AM_GNU_GETTEXT([external])
```

Comment:
Without these, running autogen fails (on a fresh checkout) with following output:
* at the beginning
```
autopoint: *** Missing version: please specify in configure.ac through a line 'AM_GNU_GETTEXT_VERSION(x.yy.zz)' the gettext version the package is using
autopoint: *** Stop.
```
* at the end
```config.status: error: cannot find input file: `po/Makefile.in.in'```

Did you remove these for a reason?

Reply:
I followed the advice from https://mail.gnome.org/archives/commits-list/2012-March/msg04458.html
GETTEXT and all i18n stuffs are blackmagic to me, but if i don't remove those 2 lines, i'm stuck with `config.status: error: po/Makefile.in.in was not created by intltoolize.`